internal/command/flags/flags.go

```diff
@@ -186,3 +186,27 @@ func AddDebugFlagsToCommand(cmd *cobra.Command, cfg *config.Config) {
 		"",
 		"The endpoint for the debug web server to listen on. It must include a port (e.g. localhost:10500).  An empty string means disable the debug endpoint.")
 }
+
+func addFirecrackerFlagsToCommand(cmd *cobra.Command, cfg *config.Config) {
+	cmd.Flags().StringVar(&cfg.FirecrackerBin,
+		firecrackerBinFlag,
+		defaults.FirecrackerBin,
+		"The path to the firecracker binary to use.")
+	cmd.Flags().BoolVar(&cfg.FirecrackerDetatch,
+		firecrackerDetachFlag,
+		defaults.FirecrackerDetach,
+		"If true the child firecracker processes will be detached from the parent flintlock process.")
+}
+
+func addCloudHypervisorFlagsToCommand(cmd *cobra.Command, cfg *config.Config) error {
+	cmd.Flags().StringVar(&cfg.CloudHypervisorBin,
+		cloudHypervisorBinFlag,
+		defaults.CloudHypervisorBin,
+		"The path to the cloud hypervisor binary to use.")
+	cmd.Flags().BoolVar(&cfg.CloudHypervisorDetatch,
+		cloudHypervisorDetachFlag,
+		true,
```

Comment:
shall we have another lil `defaults.CloudHypervisorDetach`?

Reply:
Added.